src/components/LogWindow.jsx

```diff
@@ -28,28 +28,39 @@ import '../style/logwindow.css'
 import Container from 'react-bootstrap/Container'
 import Badge from 'react-bootstrap/Badge'
 import Form from 'react-bootstrap/Form'
+import InputGroup from 'react-bootstrap/InputGroup'
 import moment from 'moment'
+import Select from 'react-select'
+import "bootstrap-icons/font/bootstrap-icons.css";
 
 
 class LogItem {
   constructor (time, status, inputPins, outputPins) {
     this.time = time;
     this.status = status;
+    this.changed = status === 'changed';
     this.inputPins = inputPins;
     this.outputPins = outputPins;
   }
 
+  // For searching/comparing
+  toString () {
+    return `[${this.time}] ${this.status} ${this.changed && '  '} | input: ${this.inputPins}, output: ${this.outputPins}`
+  }
+
   toComponent () {
     // Determine Bootstrap Badge color from status
-    let statusVariant = 'primary';
-    if (this.status === 'changed') statusVariant = 'success';
-    else if (this.status === 'unchanged') statusVariant = 'primary';
+    let statusVariant;
+    if (this.changed) statusVariant = 'success';
+    else  statusVariant = 'primary';
```

Comment:
nit: `const statusVariant = this.changed ? 'success' : 'primary'`

Reply:
no